spec/osut_tests_spec.rb

```diff
@@ -1818,7 +1818,62 @@ module M
     expect(mod1.reset(DBG)).to eq(DBG)
     expect(mod1.level).to eq(DBG)
     expect(mod1.clean!).to eq(DBG)
+    v = OpenStudio.openStudioVersion.split(".").join.to_i
 
+    p1 = OpenStudio::Point3dVector.new
+    p2 = OpenStudio::Point3dVector.new
+
+    p1 << OpenStudio::Point3d.new(3.63, 0, 4.03)
+    p1 << OpenStudio::Point3d.new(3.63, 0, 2.44)
+    p1 << OpenStudio::Point3d.new(7.34, 0, 2.44)
+    p1 << OpenStudio::Point3d.new(7.34, 0, 4.03)
+
+    t = OpenStudio::Transformation.alignFace(p1)
+
+    if v < 340
```

Comment:
OSut's `overlap` relies on the [OpenStudio.intersect](https://github.com/NREL/OpenStudio/blob/2d5ba97179292d7c836444566d0ccb34a2c18c5a/src/utilities/geometry/Intersection.cpp#L806) function to extract a potential overlapping area (i.e. a polygon) shared between 2 intersecting polygons. [TBD](https://github.com/rd2/tbd) relies on OSut's `overlap` to detect conflicts when dealing with fenestrated subsurfaces with _Frame & Divider_ frames (where widths are often fairly narrow, e.g. 3cm to 6cm). I'm unable to isolate why, but `OpenStudio.intersect` fails for intersection widths < 5cm for versions v3.2.1 through [v3.3.0](https://github.com/NREL/OpenStudio/blob/ad235ff36e046031f69023aa85551c0941f76f27/src/utilities/geometry/Intersection.cpp#L671). It works fine for versions > v3.3.0 (in fact, it works for [v3.0.0](https://github.com/NREL/OpenStudio/blob/09b7c8a554b6663a2d5e348d2d99973ea55374ad/src/utilities/geometry/Intersection.cpp#L605) as well). So here, I'm simply relaxing the test for v3.2.1 and v3.3.0, as I have no intention in developing a backwards-compatible workaround for something like 3cm frame widths.